oci/table_oci_streaming_stream.go

```diff
@@ -0,0 +1,317 @@
+package oci
+
+import (
+	"context"
+	"strings"
+
+	"github.com/oracle/oci-go-sdk/v44/common"
+	"github.com/oracle/oci-go-sdk/v44/streaming"
+	"github.com/turbot/go-kit/types"
+	"github.com/turbot/steampipe-plugin-sdk/v2/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v2/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v2/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableOciStreamingStream(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "oci_streaming_stream",
+		Description: "OCI Stream",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("id"),
```

Comment:
Please handle the not found error code if any

Reply:
Working for incorrect ids without 404 error : 

```

> select * from oci_streaming_stream where id = 'ocid1.stream.oc1'
+------+----+-----------------+--------------+-------------------------+-------------------+------------+--------------------+----------------+--------------+---------------+------+-------+--------+----------------+-----------+------+
| name | id | lifecycle_state | time_created | lifecycle_state_details | messages_endpoint | partitions | retention_in_hours | stream_pool_id | defined_tags | freeform_tags | tags | title | region | compartment_id | tenant_id | _ctx |
+------+----+-----------------+--------------+-------------------------+-------------------+------------+--------------------+----------------+--------------+---------------+------+-------+--------+----------------+-----------+------+
+------+----+-----------------+--------------+-------------------------+-------------------+------------+--------------------+----------------+--------------+---------------+------+-------+--------+----------------+-----------+------+

```